_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,72 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
```

Comment:
I don't see this anywhere else, but I think this might be a good place to reset box-sizing on everything (so we can better manage the box model) like so...

```scss
* {
    box-sizing: border-box;
}
```